site/_guides/grpc-tls-howto.md

```diff
@@ -14,7 +14,7 @@ The outcome of this is that we will have three Secrets available in the `project
 
 ### Ways you can get the certificates into your cluster
 
-- Deploy the Job from [certgen.yaml]({{ site.github.repository_url }}/tree/master/examples/contour/02-job-certgen.yaml).
+- Deploy the Job from [certgen.yaml]({{ site.github.repository_url }}/blob/{{ site.github.latest_release.tag_name }}/examples/contour/02-job-certgen.yaml).
```

Comment:
`https://github.com/projectcontour/contour/blob/v0.15.3/examples/contour/02-job-certgen.yaml` is a 404